cumulusci.properties

```diff
@@ -1,6 +1,6 @@
 cumulusci.package.name=HEDA
```

Comment:
Actually, can we delete `cumulusci.properties`?

Reply:
Well is that a question or a request ;?